ledger/src/blockstore/blockstore_purge.rs

```diff
@@ -346,97 +339,6 @@ impl Blockstore {
                 .is_ok()
     }
 
-    pub fn compact_storage(&self, from_slot: Slot, to_slot: Slot) -> Result<bool> {
```

Comment:
Confirmed that we don't need RocksDB's `compact_range` (i.e. manual compaction) as the sole purpose of the manual compaction here is to clean up obsolete entries, and this is now done by delete_files_in_range in https://github.com/solana-labs/solana/pull/26651.